eng/.docsettings.yml

```diff
@@ -72,15 +72,13 @@ known_presence_issues:
   - ['sdk/operationalinsights/Microsoft.Azure.OperationalInsights','Track 1']  
   - ['sdk/search/Microsoft.Azure.Search.Common','Track 1']
   - ['sdk/search/Microsoft.Azure.Search.Data','Track 1']
-  - ['sdk/search/Microsoft.Azure.Search.Service','Track 1']
+  - ['sdk/search/Microsoft.Azure.Search.Service','Track 1']  
   - ['sdk/operationalinsights/Microsoft.Azure.OperationalInsights','Track 1']
   - ['sdk/search/Microsoft.Azure.Search','Track 1']
   - ['sdk/search/Microsoft.Azure.Search.Common/CHANGELOG.md','Track 1']
   - ['sdk/search/Microsoft.Azure.Search.Data/CHANGELOG.md','Track 1']
   - ['sdk/search/Microsoft.Azure.Search.Service/CHANGELOG.md','Track 1']
-
-  - ['sdk/textanalytics/Azure.AI.TextAnalytics.Legacy.Shared/CHANGELOG.md','Only used for code sharing; not a shipping library']
```

Comment:
ohh interesting. I thought this was needed in order to make the build work :|

Reply:
So did I.... but apparently, if we set `<IsTestSupportLibrary>` AND `<IsShippingLibrary>` in the project file to the right things, we don't need this.